front/package.json

```diff
@@ -98,6 +98,7 @@
     "react-autosize-textarea": "^7.1.0",
     "react-csv": "2.2.2",
     "react-dates": "21.8.0",
+    "react-dnd": "^16.0.1",
     "react-dnd-cjs": "^9.5.1",
```

Comment:
To replace the SortableRow class structure (and use the new hooks) I needed to use the latest version of the `react-dnd` library. What we've been using for the last few years is[ react-dnd-cjs](https://www.npmjs.com/package/react-dnd-cjs), which essentially is an old version of the react-dnd state from 3 years ago (and it hasn't been updated since). The newest version of react-dnd is actually smaller in size, and we should really be replacing all occurrences of `react-dnd-cjs` with the new `react-dnd`, but it's used in many places in the app, so it might be better to change them later on? @EdwinKato What do you think?

Reply:
That makes sense. Let's create a ticket to remove it